### ISSUE TITLE
Error event doesn't work

### ISSUE THREAD

**Issue by darlysson** [LottieFiles/lottie-react#105]
## Overview

I would like to check if there's been an error with the animation but I'm currently unable. I've been trying it through the prop `onEvent` and even though I source a failed `.json file` I don't receive an error. 

Here's what I'm trying to do:
```
onEvent={(event) => {
        if (event === 'error') doSomenthing()
      }}
```
Any idea on how to verify that? Thank you!

## Consuming repo

> What repo were you working in when this issue occurred?

...

## Labels

- [x] Add the `Type: Bug` label to this issue.

**Comment by manojkumar3692** [comment]
the same issue, for the onEvent method only not getting trigged , 
any help?

**Comment by nattri** [comment]
Facing same issue, any fix yet?

**Comment by samuelOsborne** [comment]
Fixed with v3.5.2 🎉
